web-frontend/src/i18n/en_US.json

```diff
@@ -93,5 +93,12 @@
   "receive": {
     "account-receive": "Generate a link",
     "account-address": "Account address: "
+  },
+  "placeholder": {
+    "placeholder-banner": "Our new feature is coming soon",
```

Comment:
```suggestion
    "banner": "Our new feature is coming soon",
```

Reply:
what if we have a banner somewhere else ?

Reply:
> what if we have a banner somewhere else ?

`page.banner` & `placeholder.banner`

No problem with that

Reply:
Same name but not in the same object

Reply:
agreed, I set it to teaser-banner to be conform with the rest :)